light_dark_mode/public/index.css

```diff
@@ -0,0 +1,151 @@
+* {
+    box-sizing: border-box;
+}
+
+h1, h2, h3, h4, h5, h6, p {
+    margin: 0;
+}
+
+body {
+    margin: 0;
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+
+/* NAV styles */
+
+nav {
+    /* positioning */
+    z-index: 1;
+    /* display & box model */
+    display: flex;
+    align-items: center;
+    height: 60px;
+    padding: 30px 25px;
+    /* color */
+    background-color: #ffffff;
+    box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.25);
+}
+
+nav.dark {
+    background-color: #21222A;
+}
+
+.nav--logo_icon {
+    width: 30px;
+    margin-right: 10px;
+}
+
+.nav--logo_text {
+    color: #61DAFB;
+    margin-right: auto;
+}
+
+.nav--title {
+    color: #DEEBF8;
+}
+
+.toggler {
+    display: flex;
+    align-items: center;
+}
+
+.toggler p {
+    font-size: 0.625rem;
+    font-weight: 600;
+}
+
+.toggler--slider {
+    /* display & box model */
+    width: 25px;
```

Comment:
Please keep the same order as in nav. Flex properties should be before width and height